Android/islandRush/app/src/main/java/com/quinstedt/islandRush/MainActivity.java

```diff
@@ -10,76 +10,76 @@
 import android.view.inputmethod.EditorInfo;
 import android.widget.Button;
 import android.widget.EditText;
-import android.widget.RelativeLayout;
 import android.widget.TextView;
 import android.widget.Toast;
 
 public class MainActivity extends AppCompatActivity {
-    private EditText editText;
-    String playerNameInput;
+
+    Button enterRace, leaderboard, openCredits;
+    EditText editText;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        playerNameInput = GlobalData.getGlobalData().getPlayerData();
+
         //Dynamic background
         ConstraintLayout layout = findViewById(R.id.mainLayout);
         AnimationDrawable animationBackground = (AnimationDrawable) layout.getBackground();
         animationBackground.setEnterFadeDuration(2500);
         animationBackground.setExitFadeDuration(5000);
         animationBackground.start();
-
+       // viewPager2.findViewById(R.id.leaderboard1);
 
         // On Click goes to Controller choice
-        Button enterRace = findViewById(R.id.button_enterRace);
+        enterRace = findViewById(R.id.button_enterRace);
         enterRace.setOnClickListener(view -> openControlChoice());
-
         editText = findViewById(R.id.playerName);
-        if(!playerNameInput.isEmpty()){
-            editText.setText(playerNameInput);
-        }
+        editText.setOnEditorActionListener(editorActionListener);
 
-        /**
-         * In the MainActivity XML "android:imeOptions="actionSend" changes the Enter Button in the softKeyboard
-         * to a Send Button and this method creates a toastMessage after the Send button has been pressed in the keyboard.
-         */
 
-        editText.setOnEditorActionListener((textView, actionId, keyEvent) -> {
```

Comment:
Why are you deleting this?

Reply:
answer the question from Discord: 
There reason you are deleting stuff that shouldnt not be deleted is because the branch is behind. 
Which means that the code in the master have changed and you branch dont have those changes. 
You need to merge you android app with the master. 

if you try to merge this branch with the master you will delete the changes made to the master.

Reply:
thanks for the answer. the problem has been fixed now